crates/polars-core/src/chunked_array/temporal/mod.rs

```diff
@@ -26,6 +30,18 @@ pub fn unix_time() -> NaiveDateTime {
     NaiveDateTime::from_timestamp_opt(0, 0).unwrap()
 }
 
+#[cfg(feature = "timezones")]
+const FIXED_OFFSET_PATTERN: &str = r#"(?x)
```

Comment:
Maybe make this one static. Doesn't have to be inlined everywhere we use it I would say. (or aren't we going to use it more often?)